great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -523,6 +523,7 @@ def __init__(
         connection_string=None,
         custom_sql=None,
         schema=None,
+        data_context=None,
```

Comment:
Why do we need a data context here? If we don't need it, it shouldn't add it

Reply:
removed 🙇🏼